### PR TITLE
[MRG] Small changes

### PR DESCRIPTION
**Pull request by ShmuelTreiger**
Fixing a typo, plus attempting to clarify some documentation

### CODE REVIEW

**Review by ItsiW — APPROVED** [review]
Good changes, thanks for looking over it

File: app/backend/src/tests/test_db.py [hunk 1]
```diff
@@ -162,7 +162,7 @@ def test_migrations(testconfig):
 
     from_scratch = pg_dump()
 
-    def massage(s):
+    def message(s):
```
Comment (kalvdans) [inline app/backend/src/tests/test_db.py:165]: This was not a typo, but I realised I could have used a better verb.

Comment (ShmuelTreiger) [reply app/backend/src/tests/test_db.py:165]: Oof 🤦‍♂️ Sorry about that. Happy to fix unless you have already, just lmk

Comment (kalvdans) [reply app/backend/src/tests/test_db.py:165]: Nothing to worry about :)

**Pull request merged**